views/includes/task-list/addresses.html

```diff
@@ -2,6 +2,22 @@ <h2 class="app-task-list__section">
     Addresses
 </h2>
 <ul class="app-task-list__items">
+    {% if registeredEmailAddressOptionEnabled %}
+        <li class="app-task-list__item">
+            <span class="app-task-list__task-name">
+                <a href="{{taskList.tasks.registeredEmailAddress.url}}" data-event-id="registered-office-address-company-information-section">
+                    Registered email address
+                </a>
+            </span>
+            {% if taskList.tasks.registeredEmailAddress.state === "CHECKED" %}
+                {{ showChecked | safe | replace("statusIdPlaceholder", "roaStatus")}}
```

Comment:
I would expect the id 'roaStatus' to be replaced with 'emailStatus' or similar here (& for other 2)

Reply:
done